CRM/Contact/Task.php

```diff
@@ -111,31 +117,36 @@ public static function tasks() {
           'class' => 'CRM_Contact_Form_Task_Label',
           'result' => TRUE,
           'url' => 'civicrm/task/make-mailing-label',
+          'icon' => 'fa-print',
         ),
         self::BATCH_UPDATE => array(
-          'title' => ts('Update multiple contacts'),
+          'title' => ts('Update contacts via profile'),
```

Comment:
I'm a bit reluctant about the change of title. "update via profile" is not very meaningful to normal users.

Reply:
Ok I'll just override it on the new search UI then.